src/components/dialogs/parameters/security-analysis/security-analysis-parameters.tsx

```diff
@@ -92,15 +102,46 @@ export const SecurityAnalysisParameters: FunctionComponent<{
         },
         [snackError, updateParameters]
     );
-
     const formSchema = useMemo(() => {
-        return getLimitReductionsFormSchema(
+        const limitReductionsSchema = getLimitReductionsFormSchema(
             params.limitReductions ? params.limitReductions[0].temporaryLimitReductions.length : 0
         );
+
+        const thresholdsSchema = yup.object().shape({
+            [PARAM_SA_FLOW_PROPORTIONAL_THRESHOLD]: yup
+                .number()
+                .min(0, 'NormalizedPercentage')
+                .max(100, 'NormalizedPercentage')
+                .required(),
+            [PARAM_SA_LOW_VOLTAGE_PROPORTIONAL_THRESHOLD]: yup
+                .number()
+                .min(0, 'NormalizedPercentage')
+                .max(100, 'NormalizedPercentage')
+                .required(),
+            [PARAM_SA_LOW_VOLTAGE_ABSOLUTE_THRESHOLD]: yup.number().required(),
+            [PARAM_SA_HIGH_VOLTAGE_PROPORTIONAL_THRESHOLD]: yup
+                .number()
+                .min(0, 'NormalizedPercentage')
+                .max(100, 'NormalizedPercentage')
+                .required(),
+            [PARAM_SA_HIGH_VOLTAGE_ABSOLUTE_THRESHOLD]: yup.number().required(),
+        });
+
+        return yup.object().shape({
+            ...limitReductionsSchema.fields,
+            ...thresholdsSchema.fields,
+        });
     }, [params]);
 
     const formMethods = useForm({
-        defaultValues: { [LIMIT_REDUCTIONS_FORM]: [] },
+        defaultValues: {
+            [LIMIT_REDUCTIONS_FORM]: [],
+            [PARAM_SA_FLOW_PROPORTIONAL_THRESHOLD]: '',
+            [PARAM_SA_LOW_VOLTAGE_PROPORTIONAL_THRESHOLD]: '',
+            [PARAM_SA_LOW_VOLTAGE_ABSOLUTE_THRESHOLD]: '',
+            [PARAM_SA_HIGH_VOLTAGE_PROPORTIONAL_THRESHOLD]: '',
+            [PARAM_SA_HIGH_VOLTAGE_ABSOLUTE_THRESHOLD]: '',
```

Comment:
```suggestion
            [PARAM_SA_FLOW_PROPORTIONAL_THRESHOLD]: 0,
            [PARAM_SA_LOW_VOLTAGE_PROPORTIONAL_THRESHOLD]: 0,
            [PARAM_SA_LOW_VOLTAGE_ABSOLUTE_THRESHOLD]: 0,
            [PARAM_SA_HIGH_VOLTAGE_PROPORTIONAL_THRESHOLD]: 0,
            [PARAM_SA_HIGH_VOLTAGE_ABSOLUTE_THRESHOLD]: 0,
```

all of these data are numbers, not string
or set them to "null" if you really want to make them empty by default